apps/dashboard/components/batch-payments/index.tsx

```diff
@@ -244,35 +240,80 @@ export default function BatchPayments() {
         </>
       ) : (
         <>
           <FileUpload
+            setProcessCsvLoading={setProcessCsvLoading}
             processCsvLoading={processCsvLoading}
             file={file}
             setFile={setFile}
+            onFileProcessed={processFile}
+            resetState={resetState}
           />
-          <Card
+          <Box
             sx={{
-              margin: "1em auto",
-              padding: "1em",
+              display: "flex",
             }}
           >
-            {`Please upload a CSV file containing the following columns: "username" (the
-            recipient of the bitcoin), "dollars/sats" (this column will indicate the amount;
-            the header of this column will determine which wallet to use. If the header is
-            "dollars," your USD wallet will be utilized, and the amount will be in US dollars. If
-            "sats" is used, your BTC wallet will be employed, and the amount will be in
-            sats), and "memo" (optional)`}
-            .
-          </Card>
-          <SampleCSVTable />
+            <Card
+              sx={{
+                margin: "1em auto",
+                padding: "1em",
+                maxWidth: "60em",
+              }}
+            >
+              <Typography
+                level="h3"
+                component="div"
+                sx={{ fontSize: "h5.fontSize", mb: 2 }}
+              >
+                Fields for CSV File
+              </Typography>
+
+              <ul>
+                <li>
+                  <Typography component="div" sx={{ mb: 1 }}>
+                    <strong>Username:</strong> This column will be used to determine the
+                    Blink username of the user to whom the payment has to be sent.
+                  </Typography>
+                </li>
+                <li>
+                  <Typography component="div" sx={{ mb: 1 }}>
+                    <strong>Amount:</strong> The amount of SATS or USD you want to send.
+                    The value of sats will be converted to a whole number if not already.
+                  </Typography>
+                </li>
+                <li>
+                  <Typography component="div" sx={{ mb: 1 }}>
+                    <strong>Currency:</strong> The currency of the amount can be USD or
+                    SATS. SATS currency will only work with BTC wallets.
+                  </Typography>
+                </li>
+                <li>
+                  <Typography component="div" sx={{ mb: 1 }}>
+                    <strong>Wallet (optional):</strong> The wallet that will be used to
+                    send the payment in USD or BTC. If not provided, the default wallet
+                    will be used. If the currency is SATS and no wallet is provided, BTC
+                    wallet will be used.
+                  </Typography>
+                </li>
+                <li>
+                  <Typography component="div" sx={{ mb: 1 }}>
```

Comment:
Details about columns.

Reply:
> If the currency is SATS and no wallet is provided, BTC wallet will be used.

I don't think this is the right behavior. this is dangerous.

we should always have currency required. 

if there is a mismatch or error in what the user provides, then we should error and not process any payments

this goes how an API should generally be developed: it should be easy to use, and hard to misuse. and we want to avoid side/surprising effect.

Reply:
> > If the currency is SATS and no wallet is provided, BTC wallet will be used.
> 
> I don't think this is the right behavior. this is dangerous.
> 
> we should always have currency required.
> 
> if there is a mismatch or error in what the user provides, then we should error and not process any payments
> 
> this goes how an API should generally be developed: it should be easy to use, and hard to misuse. and we want to avoid side/surprising effect.

Currency is required. Wallet is optional; if a wallet is not provided, the default wallet will be used. But, SATS only work with a BTC wallet. Therefore, by default, BTC will be used for SATS if a wallet has not already been given.

this might be more clear 
<img width="563" alt="image" src="https://github.com/GaloyMoney/galoy/assets/59279771/a49b831e-5107-4770-88e0-0dae9b7d613c">

Reply:
>  Therefore, by default, BTC will be used for SATS if a wallet has not already been given.

my point is that I think an error here is more appropriate than attempting to use the bitcoin wallet.

Reply:
> > Therefore, by default, BTC will be used for SATS if a wallet has not already been given.
> 
> my point is that I think an error here is more appropriate than attempting to use the bitcoin wallet.

got it made changes.